py-polars/polars/expr/expr.py

```diff
@@ -5381,6 +5381,14 @@ def hash(
         seed_3
             Random seed parameter. Defaults to `seed` if not set.
 
+        Notes
+        -----
+        This implementation of :func:`rows` does not guarantee stable results
+        across different Polars versions. Its stability is only guaranteed within a
+        single version.
+        For use-cases where stability across versions is required, you can use
```

Comment:
I think it is good to add the plugin to the list of plugins, but I want to refrain from pointing to plugins in the docstrings we didn't write.

Reply:
All right will revert that part :+1: